packages/near-membrane-node/src/__tests__/error.spec.js

```diff
@@ -24,7 +24,7 @@ globalThis.foo = {
 describe('The Error Boundary', () => {
     it('should preserve identity of errors after a membrane roundtrip', () => {
         expect.assertions(3);
-        const evalScript = createVirtualEnvironment({ endowments: window });
+        const evalScript = createVirtualEnvironment({ endowments: globalThis });
```

Comment:
in node tests we were using window... which is ok because of jest, but misleading, changed that to be globalThis